test/planar.jl

```diff
@@ -3,16 +3,16 @@ module NeutralLandscapeTestPlanar
 using NeutralLandscapes
 using Test
 
-pl33 = rand(PlanarGradient(0.0), (3,3))
+pl33 = rand(PlanarGradient(0.0), 3,3)
```

Comment:
I like the old syntax as well, so we can do something like `rand(NoGradient(), size(x))` - it's also what `rand` uses. I think we should have both methods.

Reply:
OK I'll update this to have both methods

Reply:
:+1: